packages/devextreme/testing/tests/DevExpress.ui.widgets/menu.tests.js

```diff
@@ -2970,6 +2970,58 @@ QUnit.module('adaptivity: render', {
         fx.off = false;
     }
 }, () => {
+    QUnit.test('Adds "menu bar" role when menu is wider than item', function(assert) {
+        $('#qunit-fixture').width(100);
+        new Menu(this.$element, {
```

Comment:
I see that u use `width` property for updating menu width. It means that it does not matter what width `#qunit-fixture` has, it does not affect menu width while it's set through option

Reply:
[5928c7e](https://github.com/DevExpress/DevExtreme/pull/28066/commits/5928c7ef8aab472b03dce8be2f2d2f02cefed50d)